lib/quicly.c

```diff
@@ -313,6 +313,10 @@ struct st_quicly_handle_payload_state_t {
     uint64_t frame_type;
 };
 
+void __sanitizer_cov_trace_pc(void)
+{
+}
+
```

Comment:
Do we need to have this function defined in every source file? I might assume that it is required once for each executable (but I could well be wrong).

The other question is if having this symbol defined has issues when a compiler other than clang is being used. Doesn't this cause an issue when linking with other source files that also define a symbol sharing the same name (i.e. t/udpfw.c)?

Reply:
Dropping support for `trace-pc` and/or the older LLVM / clang will allow for removal of those statements, changes I describe below will do that